CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+If you want to make changes to the schema, start in the `jsonschema` directory where you can edit the JSON files.
+
+Some guidelines for making schema changes:
+
+- Use the most appropriate data type
+- Follow existing naming conventions (camelCase)
+- Add new fields last to minimise impact on consuming code
+- Probably don't add a new field as `required` - this will make the new code unable to read existing messages
+- Ideally add a description to any new fields
+
+If you need some advice, drop into the `#committers` channel on the [Cucumber Slack](https://cucumberbdd.slack.com) and ask.
```

Comment:
I prefer to link to https://cucumber.io/community#slack because that includes a sign-up link.

Reply:
Fixed on `main` - thanks.